src/test/java/org/takes/facets/auth/PsBasicTest.java

```diff
@@ -209,4 +211,63 @@ private String generateAuthenticateHead(
         );
         return String.format(AUTH_BASIC, encoded);
     }
+
+    /**
+     * Test of {@link PsBasic.Default}.
+     * @author lcozzani (lautaromail@gmail.com)
+     * @version $Id$
+     * @since 0.22
+     */
+    public static final class Default {
+
+        /**
+         * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
+         * With user and pass ok.
+         */
+        @Test
+        public void testSuccess() {
+            final String user = "Admin";
+            final String pass = "123";
+            final ConcurrentMap<String, String> usrs =
+                new ConcurrentHashMap<String, String>(1);
+            usrs.put(user, pass);
+            final Opt<Identity> identity = new PsBasic.Default(usrs)
+                .enter(user, pass);
+            MatcherAssert.assertThat(identity.has(), Matchers.is(true));
+            MatcherAssert.assertThat(
+                identity.get().urn(),
+                CoreMatchers.equalTo(PsBasicTest.generateIdentityUrn(user))
+            );
+        }
+
+        /**
+         * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
+         * With user ok and pass wrong.
+         */
+        @Test
+        public void testFailPass() {
+            final String user = "Adm";
+            final ConcurrentMap<String, String> usrs =
+                new ConcurrentHashMap<String, String>(1);
+            usrs.put(user, "124");
+            final Opt<Identity> identity = new PsBasic.Default(usrs)
+                .enter(user, "125");
+            MatcherAssert.assertThat(identity.has(), Matchers.is(false));
+        }
+
+        /**
+         * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
+         * With user wrong and pass ok.
+         */
+        @Test
+        public void testFailUser() {
```

Comment:
@lautarobock Let's call this method `returnsFalseForWrongPassword`. See [this post](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names) for a guideline.